### ISSUE TITLE
IllegalArgumentException: bound must be positive

### ISSUE THREAD

**Issue by Kim-Lan** [TheKimiNoUso/SimpleAnnouncer#1]
Hi, it doesn't seem to be causing any problems, and announcements are working well! But I've been getting this exception intermittently in my console logs

```
[SimpleAnnouncer] Task #74 for SimpleAnnouncer v1.1.1 generated an exception java.lang.IllegalArgumentException: bound must be positive
  at java.util.Random.nextInt(Random.java:322) ~[?:?]
  at me.kiminouso.simpleannouncer.AnnouncementTask.lambda$new$1(AnnouncementTask.java:35) ~[SimpleAnnouncer-1.1.1.jar:?]
  at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.19.3.jar:git-Paper-388]
  at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[paper-1.19.3.jar:git-Paper-388]
  at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1473) ~[paper-1.19.3.jar:git-Paper-388]
  at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:440) ~[paper-1.19.3.jar:git-Paper-388]
  at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1397) ~[paper-1.19.3.jar:git-Paper-388]
  at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1173) ~[paper-1.19.3.jar:git-Paper-388]
  at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-388]
  at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by TheKimiNoUso** [comment]
Hi Kim-Lan, thanks for using SimpleAnnouncer!

I've attempted to fix this for you on #2. You can download the jar here: https://github.com/TheKimiNoUso/SimpleAnnouncer/releases/tag/v1.1.2-pre

**Let me know if this fixes your issue!**

**Comment by Kim-Lan** [comment]
Looks good. I'm not seeing the error anymore, thank you!
